src/AppBundle/Entity/Post.php

```diff
@@ -179,4 +192,38 @@ public function setSummary($summary)
     {
         $this->summary = $summary;
     }
+
+    /**
+     * Add tag
+     *
+     * @param Tag $tag
+     *
+     * @return Post
+     */
+    public function addTag(Tag $tag)
+    {
+        $this->tags[] = $tag;
```

Comment:
if you don't check for duplicates here, you will get an exception when flushing because the constraint will fail in the DB